ngraph/core/src/op/convert.cpp

```diff
@@ -26,6 +26,23 @@ op::Convert::Convert(const Output<Node>& arg, const element::Type& destination_t
 void op::Convert::validate_and_infer_types()
 {
     NGRAPH_OP_SCOPE(v0_Convert_validate_and_infer_types);
+    const element::Type data_et = get_input_element_type(0);
+    const element::Type destination_et = m_destination_type;
+
+    NODE_VALIDATION_CHECK(this,
```

Comment:
Do you have a plan to add the support of LP types separately?

Reply:
Yes, I will prepare next PR with LP types.